src/pipelines/data_gen.py

```diff
@@ -195,29 +254,32 @@ def gen_observations(all_clusters, lookup_order, documentData, features, vocab,
             if 'lda' in features:
                 feature = lda(doc, corpus, vocab, lda_topics, features['lda'], feature)
 
+            if 'wordonehot' in features:
+                feature = wordonehot(doc, corpus, features['wordonehot'], feature)
```

Comment:
Believe that vocab needs to be added as a parameter in line 258 based on function definition on line 151: 
def wordonehot(doc, corpus, vocab, transformations, feature, min_length=None, max_length=None):